css/layout-helpers.css

```diff
@@ -5,3 +5,12 @@
   align-items: center;
   justify-content: center;
 }
+/*Use to caption images in EdX*/
+.figcaption {
+  font-family: Helvetica, Arial, sans-serif;
+  font-weight: normal;
+  font-style: italic;
+  font-size: 16px;
+  margin-top: 5px;
+  margin-bottom: 15px;
+}
```

Comment:
We need to put an example of how to use this in the component library - so people can see what the correct HTML is

Reply:
Example has been placed in the component library (component.html) in the FutureSkillstheme.